.taskcluster.yml

```diff
@@ -197,142 +198,156 @@ tasks:
               taskclusterProxy: true
       in:
         $match:
-          "tasks_for == 'github-pull-request' && event['action'] in ['opened', 'reopened', 'edited', 'synchronize']":
+          "(tasks_for == 'github-pull-request' && event['action'] in ['opened', 'reopened', 'synchronize']) || (tasks_for == 'github-push' && head_branch[:10] != 'refs/tags/')":
```

Comment:
I removed the `edited` action from here so that we don't retrigger the CI on Pull-requests in case we changed the title or alike.

The `github-push` section which was slightly below based on the old automation is now based on taskgraph here.